demo/build.gradle

```diff
@@ -84,6 +84,26 @@ android {
             res.srcDirs = ['./src/r2_12_1_ads/res']
             assets.srcDirs = ['./src/r2_12_1_ads/assets']
         }
+        r2_13_1 {
```

Comment:
I'm confused by this, how was 2.13.1 working without pointing to the new source before? Or was this not workign in the demo app before?

Reply:
I think it was a mistake, the demo app for 2.13 and 2.12 is not that different, we can actually use same code for 2.12, 2.13 and 2.14, but for consistancy I made a copy